rdkit/Chem/Draw/__init__.py

```diff
@@ -454,19 +452,11 @@ def _cleanupRXN(rxn):
       pass
     if compute2D:
       AllChem.Compute2DCoords(mol)
-
-  for mol in rxn.GetProducts():
-    compute2D=False
-    for conf in mol.GetConformers():
-      compute2D = not is2D(conf)
-    try:
-      AllChem.SanitizeMol(mol, 
-                          sanitizeOps=(AllChem.SanitizeFlags.SANITIZE_ALL^ 
-                                       AllChem.SanitizeFlags.SANITIZE_KEKULIZE))
-    except:
-      pass
-    if compute2D:
-      AllChem.Compute2DCoords(mol)  
+  
+def _cleanupRXN(rxn):
+  """Cleanup reactions so the computed coords are sane for depiction"""
+  map(_prepareRxnMol, rxn.GetReactants())
+  map(_prepareRxnMol, rxn.GetProducts())
```

Comment:
Well, regardless of whether it is idomatic or not, this doesn't work with python3, where map just returns an iterator. A direct loop or list comprehension is needed